Quick/QuickSpec.m

```diff
@@ -101,7 +101,11 @@ + (SEL)addInstanceMethodForExample:(Example *)example {
         [example run];
     });
     const char *types = [[NSString stringWithFormat:@"%s%s%s", @encode(id), @encode(id), @encode(SEL)] UTF8String];
-    SEL selector = NSSelectorFromString(example.name.qck_selectorName);
+    NSString *selectorName = [NSString stringWithFormat:@"%@_%@_%ld",
+                              example.name.qck_selectorName,
+                              example.callsite.file,
+                              (long)example.callsite.line];
+    SEL selector = NSSelectorFromString(selectorName);
```

Comment:
I'm curious: does this effect the test names that are output in the logs, and shown in the test navigator in Xcode?